runtime/queries/fga/highlights.scm

```diff
@@ -0,0 +1,50 @@
+(call_expression
+  function: (identifier) @function)
+
+(call_expression
+  function: (selector_expression
+    field: (identifier) @function.method))
+
+((type_identifier) @type.builtin
+  (#match? @type.builtin "^(string|int|map|uint|list|timestamp|bool|duration|double|ipaddress)$"))
```

Comment:
Should be `#any-of?` for performance reasons.

Reply:
Adjusted, thanks.